### ISSUE TITLE
Error in authStore Example for Vite

### ISSUE THREAD

**Issue by ericuldall** [marshallswain/feathers-pinia#114]
```
// stores/auth.ts
import { defineStore, acceptHMRUpdate } from 'pinia'

export const useAuthStore = defineStore('auth', () => {
  const { userStore } = useUserStore()
  const { $api } = useFeathers()

  const auth = useAuth({
    api: $api,
    userStore,
  })

  auth.reAuthenticate()

  return auth
})

if (import.meta.hot) {
  import.meta.hot.accept(acceptHMRUpdate(useAuthStore, import.meta.hot))
}

```

`const { userStore } = useUserStore()` should bue `const userStore =  useUserStore()`

**Comment by gustojs** [comment]
Same thing with Nuxt example.

**Comment by gustojs** [comment]
Actually, `$api` also needs changing, since `useFeathers` returns `api`.

```
import { defineStore, acceptHMRUpdate } from "pinia";

export const useAuthStore = defineStore("auth", () => {
  const userStore = useUserStore();
  const { api } = useFeathers();

  const auth = useAuth({
    api,
    userStore,
  });

  return auth;
});

if (import.meta.hot) {
  import.meta.hot.accept(acceptHMRUpdate(useAuthStore, import.meta.hot));
}
```

**Comment by ericuldall** [comment]
I had caught that one and changed it earlier (it was less subtle)

**Comment by marshallswain** [comment]
This is fixed in the 3.0 docs.
